ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml

```diff
@@ -413,6 +413,14 @@ objects:
           ${SSH_PUB_KEY}
         EOF
 
+        function update_image_registry() {
```

Comment:
Please move this to a separate PR, it's not related to this fix.

Reply:
Both PRs required for the test to pass.

Reply:
both commits pass minimal tests , removed the registry commit.